src/status_im/extensions/core.cljs

```diff
@@ -444,6 +444,13 @@
                  :value       :extensions/ethereum-transaction-receipt
                  :arguments   {:value     :string
                                :on-result :event}}
+                'ethereum/sign
+                {:permissions [:read]
+                 :value       :extensions/ethereum-sign
+                 :arguments   {:address?   :string
+                               :message?   :string
```

Comment:
I guess this implies we would need some way to express either this argument or this one :)

Reply:
true... when i was playing around with that i see there is a way to do something similar:

```
:arguments  {
                     :value  {:one-of #{
                                  {:message :string}
                                  {:data :string}}}
```

then we have to call

```
[ethereum/sign {:value {:data "0x617364"}}
                  :on-result [store-result]}]

[ethereum/sign {:value {:message "text"}}
                  :on-result [store-result]}]
```

are there other ways?